### PR TITLE
Create sections for gallery facet types

### PR DESCRIPTION
**Pull request by brianhelba**

### CODE REVIEW

File: web_external/ImagesGallery/Facets/ImagesFacetsPane.js [hunk 1]
```diff
@@ -35,27 +37,47 @@ const ImagesFacetsPane = View.extend({
...
-            this.$el.append(facetView.el);
-            // Do not render until the view has been inserted into the main DOM
-            facetView.render();
+        this._createFacetView('meta.clinical.diagnosis')
+            .$el.insertBefore(this.$('.isic-images-facets-clinical'));
+        this.completeFacets.forEach((completeFacet) => {
+            const facetId = completeFacet.id;
+
+            let headerEl = null;
+            // TODO: Use String.startswith and a ES6 polyfill
```
Comment (msmolens) [inline web_external/ImagesGallery/Facets/ImagesFacetsPane.js]: Does `string.startsWith` not work?

Comment (brianhelba) [reply web_external/ImagesGallery/Facets/ImagesFacetsPane.js]: It's a new feature of ES6. We don't provide a polyfill at the moment, so there's no reason to break unnecessarily until we add a polyfill or explicitly drop support for browsers without native ES6 support.

Comment (msmolens) [reply web_external/ImagesGallery/Facets/ImagesFacetsPane.js]: I see, even with Babel using this would require a polyfill.

Comment (brianhelba) [reply web_external/ImagesGallery/Facets/ImagesFacetsPane.js]: Yeah.

**Pull request merged**